index.html

```diff
@@ -4,8 +4,109 @@
     <meta charset="UTF-8" />
     <title>Fork Me! FCC: Test Suite Template</title>
     <link rel="stylesheet" href="css/style.css" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
   </head>
 
-  <body></body>
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <header id="header" class="header">
+      <img
+        id="header-img"
+        src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"
+        alt="logo"
+      />
+      
+      <nav id="nav-bar">
+        <a href="#features" class="nav-link">Features</a>
+        <a href="#how-it-works" class="nav-link">How It Works</a>
+        <a href="#pricing" class="nav-link">Pricing</a>
+      </nav>
+    </header>
+
+    <section id="heroSection">
+      <h2 class="heroSection_title">Handcrafted, home-made masterpieces</h2>
+
+      <form id="form" action="https://www.freecodecamp.com/email-submit">
+        <div class="form-container">
+          <input type="email" id="email" placeholder="Type your email" name="email" />
+          <input type="submit" id="submit" />
+        </div>
+      </form>
+
+      <div id="features">
+        <p class="firstParagraph"><h3>Premium Materials</h3>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.</p>
+        <p class="firstParagraph"><h3>Fast Shipping</h3>We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied.</p>
+        <p class="firstParagraph"><h3>Quality Assurance</h3>For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument.</p>
+      </div>
+
+      <section id="how-it-works">
+        <iframe id="video" src="https://www.youtube.com/embed/y8Yv4pnO7qc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+      </section>
+
+      <section id="pricing">
+        <div class="pricing_plan">
+          <div class="pricing_plan_head">
+            TENOR TROMBONE
+          </div>
+<!--test-->
+          <div class="pricing_plan_price">$500</div>
+
+          <div class="pricing_plan_body">
+            <p>Lorem Ipsum.</p>
+            <p>Lorem Ipsum.</p>
+            <p>Lorem Ipsum.</p>
+            <p>Lorem Ipsum.</p>
+          </div>
```

Comment:
This is expected to be a list item li and should be under an unordered list ul
Please do that for all the cards and group them together